apps/system/utils/utils_cpuload.c

```diff
@@ -62,6 +62,14 @@
 #define CPULOAD_BUFLEN 128
 #define CPULOADMON_RUNNING_FOREVER -1
 
+#ifdef CONFIG_SCHED_MULTI_CPULOAD
+static volatile uint32_t cpuload_timeconstant[SCHED_NCPULOAD] = {
+	CONFIG_SCHED_CPULOAD_TIMECONSTANT_SHORT,
+	CONFIG_SCHED_CPULOAD_TIMECONSTANT_MID,
+	CONFIG_SCHED_CPULOAD_TIMECONSTANT_LONG
+};
```

Comment:
Does this need to be a global variable? Can we move it into the function where its being used?